quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java

```diff
@@ -80,6 +81,24 @@ void testMetricsEndpointWithCacheMetricsHistograms() {
 
     }
 
+    @Test
+    @Launch({ "start-dev", "--metrics-enabled=true", "--features=user-event-metrics", "--user-event-metrics-enabled=true" })
+    void testMetricsEndpointWithUserEventMetrics(KeycloakDistribution distribution) {
```

Comment:
It would be good to also test that the user event metrics are not presented when disabled.